src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -122,6 +122,22 @@ public void FlyWithWiresA(double starboard, double port){
     portMotor.set(ControlMode.PercentOutput, port);
   }
 
+  /**
+   * Drives the robot using two values for each wheel
+   * in velocity output (m/s)
+   * @author Joe Sedutto
+   * @param starboard_meters (A Value in M/s)
+   * @param port_meters (A value in M/s)
+   */
+  public void FlyWithWiresB(double starboard_meters, double port_meters){
+    // Convert the M/s into M/10ms, then, convert M/10ms into Ticks/10ms
+    double starboard_miliseconds = ((starboard_meters / 100) * Constants.DrivetrainPID.ticksPerMeter);
+    double port_miliseconds = ((port_meters / 100) * Constants.DrivetrainPID.ticksPerMeter);
+
+    starboardMotor.set(ControlMode.Velocity, starboard_miliseconds);
+    portMotor.set(ControlMode.Velocity, port_miliseconds);
+  }
+
```

Comment:
I thought the RIO is not this precise?

Reply:
Im not sure what you mean by that, these are being handed to the motor controllers.

Reply:
My bad

Reply:
No worries :)

Reply:
This looks good. I see no issues